src/controllers/controller.doughnut.js

```diff
@@ -145,18 +145,16 @@ module.exports = DatasetController.extend({
 		var chart = me.chart;
 		var chartArea = chart.chartArea;
 		var opts = chart.options;
-		var availableWidth = chartArea.right - chartArea.left;
-		var availableHeight = chartArea.bottom - chartArea.top;
-		var minSize = Math.min(availableWidth, availableHeight);
+		var size = {width: 1, height: 1};
```

Comment:
If we don't need to return or manipulate `size` as an object, I would use `var width = 1; var height = 1` directly.